app/services/catalog/copy_portfolio_item.rb

```diff
@@ -24,8 +24,8 @@ def process
 
     def make_copy
       @portfolio_item.dup.tap do |new_portfolio_item|
-        new_portfolio_item.name = new_name(@portfolio_item.name, :name)
-        new_portfolio_item.display_name = new_name(@name, :display_name)
+        new_portfolio_item.name = @params[:portfolio_item_name] || new_name(@portfolio_item.name, :name)
+        new_portfolio_item.display_name = @params[:portfolio_item_name] || new_name(@portfolio_item.display_name, :display_name)
```

Comment:
@lindgrenj6 
Should this be
```
new_portfolio_item.display_name = new_name(@portfolio_item.display_name, :display_name) || @params[:portfolio_item_name]
```
Otherwise it will always use the name, if there is a display_name we should use that first, if its empty then generate a new name.